demo/.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+.next/
+out/
+dist/
```

Comment:
Extra line here :+1:

Reply:
There was already an extra line, but GH seems to kinda ignore it. See in the template https://github.com/moxystudio/react-lib-template/blob/master/demo/.gitignore , there doesn't seem to be an extra line at the end as well. How should we proceed?